tests/emerge_ebuild.sh

```diff
@@ -13,7 +13,8 @@ EBUILD="${1}"
 echo "Emerging ${EBUILD}"
 
 # Disable news messages from portage and disable rsync's output
-export FEATURES="-news" PORTAGE_RSYNC_EXTRA_OPTS="-q"
+export FEATURES="-news" \
```

Comment:
for bash it is but for us humans we it is in multiple lines so we can read it better.